ui_curses.c

```diff
@@ -1559,7 +1562,7 @@ void info_msg(const char *format, ...)
 	va_list ap;
 
 	va_start(ap, format);
-	vsnprintf(error_buf, sizeof(error_buf), format, ap);
+	vsnprintf(error_buf, sizeof(error_buf), _(format), ap);
```

Comment:
the parameter to info_msg should be localize at the caller. i.e. `info_msg(_("%s not found: %s"), what, search_str ? search_str : "");`

Reply:
Oh, that's defintely a mistake on my part. I missed that call when I added info_msg to gettext. So what should I undo? Translating info_msg or the double accidental translation?

Reply:
Alright, this should all be fixed. Now each call of these functions is translated in place.